### DISCUSSION TITLE
Context/Hook based Flagsmith integration for React apps

### DISCUSSION THREAD

**Discussion by TheMagoo73** [Flagsmith/flagsmith#78] [Show and tell] [answered]
As part of a new React project I'm working on we wanted to integrate Flagsmith as our feature flagging tool. I'm a big fan of context and hooks in React, especially for something like feature flags which many components might want to access. To that end I've started working on a context and hook based integration for Flagsmith based around the existing Javascript integration.

It's pretty rough and ready at the moment as it's early days, but we've integrated it into out POC and it's functional. I'm hoping to get a properly stable V1 out in the next week or so.

Any ideas, feedback or bugs would be gratefully accepted - I hope it's of use to people.

- [GitHub](https://github.com/TheMagoo73/flagsmith-react)
- [NPM](https://www.npmjs.com/package/flagsmith-react)

**Comment by kyle-ssg** [comment, marked as answer]
Excited to use this! I'll keep an eye on the repo and test / provide any insights when I can.

**Reply by TheMagoo73** [reply]
Cheers @kyle-ssg - I've just tagged version 1.0.0 which I think is suitable for production use. Feedback always welcome!
